src/main/java/frc/robot/subsystems/climber/ClimberIOSparkMax.java

```diff
@@ -0,0 +1,88 @@
+package frc.robot.subsystems.climber;
+
+import static frc.robot.Constants.ClimberConstants.*;
+
+import com.revrobotics.CANSparkBase.ControlType;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkPIDController;
+import com.revrobotics.SparkPIDController.ArbFFUnits;
+import edu.wpi.first.math.util.Units;
+import frc.robot.subsystems.climber.ClimberIO.ClimberIOInputs;
+
+/**
+ * NOTE: To use the Spark Flex / NEO Vortex, replace all instances of "CANSparkMax" with
+ * "CANSparkFlex".
+ */
+public class ClimberIOSparkMax implements ClimberIO {
+  private static final double GEAR_RATIO = 1.5;
+
+  private final CANSparkMax m_winchMotorRight = new CANSparkMax(10, MotorType.kBrushless);
+  private final CANSparkMax m_winchMotorLeft = new CANSparkMax(11, MotorType.kBrushless);
+  private final RelativeEncoder encoder = m_winchMotorRight.getEncoder();
+  private final SparkPIDController pid = m_winchMotorRight.getPIDController();
+
+  public ClimberIOSparkMax() {
+    m_winchMotorRight.restoreFactoryDefaults();
+    m_winchMotorLeft.restoreFactoryDefaults();
+
+    m_winchMotorRight.setIdleMode(CANSparkMax.IdleMode.kBrake);
+    m_winchMotorLeft.setIdleMode(CANSparkMax.IdleMode.kBrake);
+
+    m_winchMotorRight.setSmartCurrentLimit(extendCurrentLimit);
+    m_winchMotorLeft.setSmartCurrentLimit(
+        extendCurrentLimit); // make sure to have logic for homing current limit
+
+    m_winchMotorRight.setInverted(false);
+    m_winchMotorLeft.setInverted(false);
+
+    m_winchMotorRight.setSmartCurrentLimit(extendCurrentLimit);
+    m_winchMotorLeft.setSmartCurrentLimit(extendCurrentLimit);
+
+    m_winchMotorRight.burnFlash();
```

Comment:
I don't think you need to burn flash every time. They only have ~10K flashes or so.

What might be useful is a comparison function like "if the parameters we are setting don't match the flash, burn flash". So that way if a spark max gets swapped you ensure after a brownout that its still in brake or idle mode, for example.